maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm

```diff
@@ -0,0 +1,233 @@
+//Parent of this place is kara.dm
+// -- Datums -- //
+
+/datum/map_template/sc_lateload/away_aerostat
+	name = "Kara Aerostat - Z1 Aerostat"
+	desc = "The Kara Aerostat away mission."
+	mappath = 'maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dmm'
+	associated_map_datum = /datum/map_z_level/sc_lateload/away_aerostat
+
+/datum/map_z_level/sc_lateload/away_aerostat
+	name = "Away Mission - Aerostat"
+	z = Z_LEVEL_AEROSTAT
+
+
+// -- Overmap -- //
+
+/obj/effect/overmap/visitable/sector/aerostat
+	name = "Aerostat"
+	desc = "Abandoned aerostat installation, floating in the low stratosphere."
+	scanner_desc = @{"[i]Registration[/i]: Vir System Authority
+[i]Class[/i]: Installation
+[i]Transponder[/i]: Transmitting (CIV), Vir IFF
+[b]Notice[/b]: CONDEMNED! NO ENTRY! -Vir System Authority"}
+	initial_generic_waypoints = list("aerostat_west","aerostat_east","aerostat_south","aerostat_northwest","aerostat_northeast")
+	icon_state = "object"
+	unknown_state = "object"
+	known = TRUE
+	start_x =  11
+	start_y =  11
+//	extra_z_levels = list(Z_LEVEL_AEROSTAT_2) //SOON
+
+
+// -- Areas -- //
+
+/area/sc_away/aerostat
+	name = "\improper Away Mission - Aerostat Outside"
+	icon_state = "away"
+	base_turf = /turf/unsimulated/floor/sky/kara_sky
+	requires_power = FALSE
+	dynamic_lighting = FALSE
+	flags = RAD_SHIELDED
+
+/area/sc_away/aerostat/inside
+	name = "\improper Away Mission - Aerostat Inside"
+	icon_state = "crew_quarters"
+	base_turf = /turf/simulated/floor/plating/kara
+	requires_power = TRUE
+	dynamic_lighting = TRUE
+	forced_ambience = list('sound/ambience/tension/tension.ogg', 'sound/ambience/tension/argitoth.ogg', 'sound/ambience/tension/burning_terror.ogg')
+
+/area/sc_away/aerostat/solars
+	name = "\improper Away Mission - Aerostat Solars"
+	icon_state = "crew_quarters"
+	base_turf = /turf/simulated/floor/plating/kara
+	dynamic_lighting = TRUE
+
+
+// -- Landmarks -- //
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_west
+	name = "Aerostat - West Dock"
+	landmark_tag = "aerostat_west"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_east
+	name = "Aerostat - East Dock"
+	landmark_tag = "aerostat_east"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_south
+	name = "Aerostat - South Dock"
+	landmark_tag = "aerostat_south"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_northwest
+	name = "Aerostat - Northwest Dock"
+	landmark_tag = "aerostat_northwest"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_northeast
+	name = "Aerostat - Northeast Dock"
+	landmark_tag = "aerostat_northeast"
+
```

Comment:
Literally all of the above should be purely reflective of the fuel depot, but the aerostat is not behaving like the fuel depot:
1. It is not appearing on the overmap
2. It is not appearing on sensors for the Baby Mammoth
3. It is not appearing as an autopilot target
4. The aerostat west dock is not appearing as a docking target.

WHY

Reply:
Are the landmarks placed on the Aerostat map for each dock, and is the Visitable landmark placed on the Aerostat map?